try.html

```diff
@@ -0,0 +1,766 @@
+        }
+        .loader {
+          font-size: 10px;
+          position: absolute;
+          top: 80px;
+          left: calc(50% - 5em);
+          z-index:1000;
+          display:none;
+
+          font-size: 10px;
+          text-indent: -9999em;
+          border-top: 1.1em solid rgba(0,87,155, 0.2);
+          border-right: 1.1em solid rgba(0,87,155, 0.2);
+          border-bottom: 1.1em solid rgba(0,87,155, 0.2);
+          border-left: 1.1em solid #00579b;
+          -webkit-transform: translateZ(0);
+          -ms-transform: translateZ(0);
+          transform: translateZ(0);
+          -webkit-animation: load8 1.1s infinite linear;
+          animation: load8 1.1s infinite linear;
+        }
+        @-webkit-keyframes load8 {
+          0% {
+            -webkit-transform: rotate(0deg);
+            transform: rotate(0deg);
+          }
+          100% {
+            -webkit-transform: rotate(360deg);
+            transform: rotate(360deg);
+          }
+        }
+        @keyframes load8 {
+          0% {
+            -webkit-transform: rotate(0deg);
+            transform: rotate(0deg);
+          }
+          100% {
+            -webkit-transform: rotate(360deg);
+            transform: rotate(360deg);
+          }
+        }
+
+        .tooltip-inner {
+            max-width:600px;
+            text-align:left;
+            font-size: 14px;
+        }
+    </style>
+
+</head>
+
+<body id="top">
+
+<div class="navbar navbar-inverse navbar-fixed-top">
+    <div class="navbar-inner">
+        <div class="container">
+            <h1><a href="#top">jQuery Mapael</a></h1>
+            <ul class="nav">
+                <li><a href="/mapael/#overview">Overview</a></li>
+                <li><a href="/mapael/#examples">Examples</a></li>
+                <li><a href="/mapael/#api-reference">Documentation</a></li>
+                <li><a href="https://github.com/neveldo/mapael-maps" target="_blank">Additional maps</a></li>
+                <li><a href="create-map.php">Create your own map</a></li>
+                <li><a href="https://twitter.com/VincentBroute" target="_blank"><img src="assets/img/twitter_logo.png" width="18px" height="15px" style="vertical-align:top;" /> Twitter</a></li>
+                <li><a href="https://github.com/neveldo/jQuery-Mapael" target="_blank"><img src="assets/img/github_logo.png" width="15px" height="15px" style="vertical-align:top;" /> Github</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <h2 style="margin-bottom:10px;text-align:center;">Try jQuery Mapael online</h2>
+
+
+    <div class="row">
+
+        <!-- Map url / file / CSV pannel -->
+        <div class="col-sm-6">
+
+            <div class="form-group">
+
+                <div class="row">
+                    <div class="col-sm-4">
+                        <label for="mapSelector">Map</label>
+                        <select class="form-control" id="mapSelector" name="mapSelector"></select>
```

Comment:
Clicking on the Map label seems to trigger the focus on the select, but it would be best if it opens it as well?